targets/TARGET_STMBLUE/hal/inc/bluenrg_x_device.h

```diff
@@ -0,0 +1,97 @@
+/**
+  ******************************************************************************
+  * @file    BlueNRG_x_device.h
+  * @author  VMA Application Team
+  * @version V1.0.0
+  * @date    23-January-2017
+  * @brief   This file is used to select the BlueNRG-1 or BlueNRG-2 device.
+  ******************************************************************************
+  * @attention
+  *
+  * THE PRESENT FIRMWARE WHICH IS FOR GUIDANCE ONLY AIMS AT PROVIDING CUSTOMERS
```

Comment:
Can this driver be released under Apache 2.0 ? I dont see any license in this header file or similar

Reply:
Do you refer to this `bluenrg_x_device.h` or more in general to all files with this header? It is the original header included in BlueNRG-2 Development Kit,. Files with this header inside the target folder are related to `libbluenrg1_stack.a` static library that is covered by STMicroelectronics SLA0068.

Concerning files under `TARGET_STMBLUE` path, implementing mbed os API, I asked ST if it is possible to release under Apache 2.0.

Reply:
> Do you refer to this `bluenrg_x_device.h` or more in general to all files with this header? It is the original header included in BlueNRG-2 Development Kit,. Files with this header inside the target folder are related to `libbluenrg1_stack.a` static library that is covered by STMicroelectronics SLA0068.

yes, the license should follow https://os.mbed.com/docs/mbed-os/v5.11/contributing/license.html

In this case, `libbluenrg1_stack` lib to be released under Permissive Binary License